airscan.go

```diff
@@ -328,6 +382,10 @@ func (c *Client) Scan(settings *ScanSettings) (*ScanState, error) {
 	}, nil
 }
 
+func (c *Client) GetEndpoint(s string) string {
```

Comment:
GetEndpoint is still exported. Does it need to be? Could you unexport it for now please?

Reply:
You're right, we don't need to export it! Sorry